### PR TITLE
Nc gj feature graphcool server

### PR DESCRIPTION
**Pull request by nancychuchu**
removed resolvers and hooks to implement avgRating on server and instead, added a helper method on the client side.

### CODE REVIEW

File: server/graphcool.yml [hunk 1]
```diff
@@ -37,22 +30,21 @@ permissions:
...
-  loggedInUser:
-    type: resolver
-    schema: src/email-password/loggedInUser.graphql
-    handler:
-      code: src/email-password/loggedInUser.ts
+#   loggedInUser:
+#     type: resolver
+#     schema: src/email-password/loggedInUser.graphql
+#     handler:
+#       code: src/email-password/loggedInUser.ts
```
Comment (siddharthparmar7) [inline server/graphcool.yml]: Do you want to remove these functions?

File: server/src/avgRating.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+export default async event => {
+  console.log(event);
+
+  try {
+    //Create Graphcool API (based on https://github.com/graphcool/graphql-request)
+    const graphcool = fromEvent(event);
+    const api = graphcool.api("simple/v1");
+
+    const lockerId = event.data.id;
+    // const avgRating = await getAvgRating(api, lockerId);
```
Comment (siddharthparmar7) [inline server/src/avgRating.js]: Please remove commented code.

File: server/src/startRating.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// console.log("heyo");
+// updateOrCreateReview(update: UpdateReview!, create: CreateReview!): Review
```
Comment (siddharthparmar7) [inline server/src/startRating.js]: Please remove unnecessary comments. Thank you.

File: server/src/startRating.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+export default event => {
+  console.log(event.data);
```
Comment (siddharthparmar7) [inline server/src/startRating.js]: remove this console log.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: server/graphcool.yml (hunk 1 above)
Comment (nancychuchu) [reply server/graphcool.yml]: Nope. We need these. They become uncommented in the NC-feature-Auth branch

File: server/src/avgRating.js (hunk 2 above)
Comment (nancychuchu) [reply server/src/avgRating.js]: These entire files were supposed to be deleted. I don't know how they came back to life. They've been deleted again.

File: server/src/startRating.js (hunk 3 above)
Comment (nancychuchu) [reply server/src/startRating.js]: These entire files were supposed to be deleted. I don't know how they came back to life. They've been deleted again.

File: server/src/startRating.js (hunk 4 above)
Comment (nancychuchu) [reply server/src/startRating.js]: These entire files were supposed to be deleted. I don't know how they came back to life. They've been deleted again.

File: server/graphcool.yml (hunk 1 above)
Comment (siddharthparmar7) [reply server/graphcool.yml]: We need them in this branch too. Please uncomment these lines.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**